### DISCUSSION TITLE
New wallpapers or new Global themes will not install

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#322] [General]
Getting similar error message in both cases:

Download of "Uzuri v2 Dark" failed, error: Error transferring https://files03.pling.com/api/files/download/j/eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6IjE2OTY5NTcwMjUiLCJvIjoiMSIsInMiOiI2NThjMGY2YzkzZWUxYTg1YmIyMGQxODViNTI2NTU1OGFjOWU5OTE1ZTJkMTczNjVmYjQ1OWFhZmU1Y2U1ZmRlMWVkYjg5ZDU5ZTIzMDhkOWM4N2VlMWNiMTI2OTBkZjM2NTJmMzVhNGNhZDQ0MTAxODBiOWRiMTliODU3MzdlMyIsInQiOjE2OTY5NzQ3NjcsInN0ZnAiOm51bGwsInN0aXAiOiIyMDYuMjU1LjE5Mi4xOTEifQ.vOFaDyWXwiD_gd-wq_nWFAPxnQFTRtL8Olzp4u36fGw/Uzuri-v2-Dark.tar.xz - server replied:

- - - - -

OS: Debian GNU/Linux 12 (bookworm) x86_64 
 ,$$P'              `$$$.     Host: OptiPlex 7060 
',$$P       ,ggs.     `$$b:   Kernel: 6.4.0-0.deb12.2-amd64 
`d$$'     ,$P"'   .    $$$    Uptime: 14 mins 
 $$P      d$'     ,    $$P    Packages: 1910 (dpkg), 1 (flatpak) 
 $$:      $$.   -    ,d$$'    Shell: bash 5.2.15 
 $$;      Y$b._   _,d$P'      Resolution: 1920x1080, 1920x1080 
 Y$$.    `.`"Y$$$$P"'         DE: Plasma 5.27.5 
 `$$b      "-.__              WM: KWin 
  `Y$$                        Theme: Numix [GTK2/3] 
   `Y$$.                      Icons: breeze-dark [GTK2/3] 
     `$$b.                    Terminal: konsole 
       `Y$$b.                 CPU: Intel i5-8500 (6) @ 4.100GHz 
          `"Y$b._             GPU: Intel CoffeeLake-S GT2 [UHD Graphics 630] 
              `"""            Memory: 3334MiB / 7736MiB

**Comment by geckolinux** [comment]
That's a common problem in KDE, it's an issue with their servers.

**Comment by dacomboman** [comment]
Hopefully a temporary one.
